janus_server/src/trace.rs

```diff
@@ -32,6 +43,12 @@ pub struct TraceConfiguration {
     /// (optional)
     #[serde(default)]
     pub tokio_console_config: TokioConsoleConfiguration,
+    /// Enable delivering OpenTelemetry traces to a Jaeger agent.
```

Comment:
I think we should document that `otel_jaeger` & `otel_otlp` are mutually-exclusive.

(Better yet would be making them mutually exclusive at the data structure level, e.g. similar to a Rust enum, but I'm not sure YAML encoding supports this kind of "oneof" functionality.)

Reply:
We could put these two in their own enum, serde would turn that into another layer of nesting, like so:

```yaml
logging_config:
  otel:
    otlp:
      enabled: true
      endpoint: "https://www.example.com"
```

When parsing the enum, serde would complain loudly if it found a dictionary with both "jaeger" and "otlp" keys, rather than one or the other.